frontend/src/component/menu/routes.ts

```diff
@@ -518,24 +518,7 @@ export const routes: IRoute[] = [
 export const getRoute = (path: string) =>
     routes.find((route) => route.path === path);
 
-export const baseRoutes = routes.filter((route) => !route.hidden);
-
-const computeRoutes = () => {
-    const mainNavRoutes = baseRoutes.filter((route) => route.menu.advanced);
-    const adminRoutes = routes.filter((route) => route.menu.adminSettings);
-    const mobileRoutes = routes.filter((route) => route.menu.mobile);
```

Comment:
Out of all these - only the mainNavRoutes were being used (mobile no longer in use, and admin has it's own adminRoutes.ts). No point returning this object